Control-Systems/Control-Systems.md

```diff
@@ -0,0 +1,93 @@
+# Control Systems
+A control system is defined as a system of devices that manages, commands, directs, or regulates the behavior of other devices or systems to achieve a desired result.
+It achieves this through control loops, which are a process designed to maintain a process variable at a desired set point.
+The three basic parts of a control system are:  
+  * The error detector
+  * The controller 
+  * The output element
+
+### Open and Closed Loop Control Systems
+Control Systems can be divided into Open-loop and Closed-loop control systems. 
+In open loop control systems, output is not fed-back to the input. So, the control action is independent of the desired output. These systems are easier to design but have less accuracy because there is no feedback from the output.
+Examples of open loop control systems include traffic lights, immersion water heaters etc.
+![image](https://user-images.githubusercontent.com/72137415/148272259-407e6466-83d0-4a4a-bd96-34cf7c0c47fc.png)
+
+In closed loop control systems, output is fed back to the input. So, the control action is dependent on the desired output. These systems are more complex and difficult to design but have better control over the output because of the feedback and hence the output is accurate. 
+Examples of such systems include air conditioner, pressure control system etc. 
+![image](https://user-images.githubusercontent.com/72137415/148272214-2af134d9-4b9e-4313-8c27-539c8486f78f.png)
+
+### Transfer Function
+A transfer function represents the relationship between the output signal of a control system and the input signal, for all possible input values.
+In a Laplace Transform, if the input is represented by R(s) and the output is represented by C(s), then the transfer function will be: 
+![image](https://user-images.githubusercontent.com/72137415/148949693-fe4092f8-df35-4be8-a756-7f940c17aa7b.png)
+
+### Representation of Control Systems
+Control systems are represented in two main ways, which are as follows:
+   * Block Diagrams : Here, a control system is represented by blocks to depict it pictorially. Block diagrams contain blocks (which represent the transfer function and have one input and one output), take-off point (representing branching of signals) and summation points (which depict an algebraic summation of two or more input signals to one output signal).
+   ![image](https://user-images.githubusercontent.com/72137415/148892599-cfa0f187-9c33-4378-ac39-333fe9a0d379.png) \
```

Comment:
Leave a gap between the image and text, the formatting at this place looks odd.
![image](https://user-images.githubusercontent.com/53267275/149310713-b4e73f92-bff8-4a91-b24d-1ee83ff0bb99.png)